src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -876,7 +876,7 @@ namespace winrt::TerminalApp::implementation
 
         if (debugConnection) // this will only be set if global debugging is on and tap is active
         {
-            TermControl newControl{ settings, debugConnection };
+            TermControl newControl{ *(winrt::get_self<TerminalSettings>(settings)->CreateChild()), debugConnection };
```

Comment:
This change doesn't affect the settings reload bug but I figured we should update this call as well

Reply:
It actually will impact reload, if the pane is a debug tap pane :D